pkgs/development/libraries/gtk+/3.x.nix

```diff
@@ -21,12 +21,14 @@ stdenv.mkDerivation rec {
     sha256 = "1l45nd7ln2pnrf99vdki3l7an5wrzkbak11hnnj1w6r3fkm4xmv1";
   };
 
+  NIX_LDFLAGS = stdenv.lib.optionalString stdenv.isDarwin "-lintl";
```

Comment:
No worries! Happy to make the change. I'll wait for feedback from @shlevy though, because I'm interested to know for sure.